src/ex1/script.js

```diff
@@ -0,0 +1,284 @@
+//Selectors
+const todoInput = document.getElementById("input-txt");
+const todoButton = document.getElementById("add-btn");
+const todoList = document.getElementById("list-element");
+const clearAllBtn = document.getElementById("clearAll-btn");
+const sortBtn = document.getElementById("sort-btn");
+const inputBox = document.querySelector(".inputField input");
+const svgContainer = document.getElementById("svg");
+const todoElement = document.getElementById("list-element");
```

Comment:
All of the logic of getElement / querySelector should be after document ready event to ensure the above elements are actually exist on the DOM at the time of query